dashboard/app/views/layouts/_google_theme_analytics.html.haml

```diff
@@ -0,0 +1,20 @@
+-# The following dimensions must match in order the custom dimension settings in our Google Analytics account.
```

Comment:
We could take this opportunity to move all of this code into a single `:ruby` block.